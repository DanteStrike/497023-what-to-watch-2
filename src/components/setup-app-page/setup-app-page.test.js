import React from "react";
import renderer from "react-test-renderer";
import SetupAppPage from "./setup-app-page.jsx";


it(`Render correctly SetupAppErrorPage component`, () => {
  const setupAppErrorMock = {
    isError: true,
    code: 400,
    msg: `Any`,
  };
  const component = renderer
    .create(
        <SetupAppPage setupAppError={setupAppErrorMock} onRepeatSetupClick={jest.fn()}/>
    ).toJSON();

  expect(component).toMatchSnapshot();
});
