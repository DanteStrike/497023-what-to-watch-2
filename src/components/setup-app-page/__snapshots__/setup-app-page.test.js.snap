// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render correctly SetupAppErrorPage component 1`] = `
<div
  className="user-page"
>
  <div
    className="user-page__content"
    style={
      Object {
        "display": "flex",
      }
    }
  >
    <div
      style={
        Object {
          "margin": "auto",
        }
      }
    >
      <h1>
        Something goes wrong on setup App
      </h1>
      <p>
        Error code: 
        400
      </p>
      <p
        style={
          Object {
            "marginBottom": "60px",
          }
        }
      >
        Any
      </p>
      <button
        className="btn"
        onClick={[MockFunction]}
        style={
          Object {
            "border": "1px solid rgba(217,205,141,.41)",
            "borderRadius": "8px",
            "color": "#d9cd8d",
            "cursor": "pointer",
            "fontSize": "30px",
            "lineHeight": "44px",
            "margin": "auto",
            "textAlign": "center",
            "transition": "border-color .2s",
          }
        }
      >
        Retry
      </button>
    </div>
  </div>
</div>
`;
